t/selenium/Redirect_Merged_Entities.html

```diff
@@ -0,0 +1,178 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
+<head profile="http://selenium-ide.openqa.org/profiles/test-case">
+<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+<link rel="selenium.base" href="http://localhost:5000" />
+<title>Redirect_Merged_Entities</title>
+</head>
+<body>
+<table cellpadding="1" cellspacing="1" border="1">
+<thead>
+<tr><td rowspan="1" colspan="3">Redirect_Merged_Entities</td></tr>
+</thead><tbody>
+<!--Artist tests (default redirects, actions, GET parameters and two actions)-->
+<tr>
+	<td>open</td>
+	<td>/artist/f21a407e-3af9-4539-ab3d-c92a5230dff6</td>
+	<td></td>
+</tr>
+<tr>
+	<td>assertLocationMatches</td>
+	<td>\/artist\/1155431a-d35e-4863-9ae0-e3c24eb61aa9</td>
+	<td></td>
+</tr>
+<tr>
+	<td>open</td>
+	<td>/artist/f21a407e-3af9-4539-ab3d-c92a5230dff6/releases</td>
+	<td></td>
+</tr>
+<tr>
+	<td>assertLocationMatches</td>
+	<td>\/artist\/1155431a-d35e-4863-9ae0-e3c24eb61aa9\/releases</td>
+	<td></td>
+</tr>
+<tr>
+	<td>open</td>
+	<td>/artist/f21a407e-3af9-4539-ab3d-c92a5230dff6/recordings?filter.name=do</td>
+	<td></td>
+</tr>
+<tr>
+	<td>assertLocationMatches</td>
+	<td>\/artist\/1155431a-d35e-4863-9ae0-e3c24eb61aa9\/recordings\?filter.name=do</td>
+	<td></td>
+</tr>
+<tr>
+	<td>open</td>
+	<td>/artist/f21a407e-3af9-4539-ab3d-c92a5230dff6/alias/37382/edit</td>
+	<td></td>
+</tr>
+<tr>
+	<td>assertLocationMatches</td>
+	<td>\/artist\/f21a407e-3af9-4539-ab3d-c92a5230dff6\/alias\/37382\/edit</td>
```

Comment:
Should be:

```suggestion
	<td>\/artist\/1155431a-d35e-4863-9ae0-e3c24eb61aa9\/alias\/37382\/edit</td>
```

Otherwise it's testing that it doesn't redirect. The reason it passes now is that it's an edit page and you need to be logged-in to view those, so it immediately hits a login screen. You can specify that the test should run while logged-in in t/selenium.js.

Reply:
Whoops...